crates/dkg-cli/src/actions.rs

```diff
@@ -333,11 +344,13 @@ async fn wait_for_phase<P: JsonRpcClient, S: Signer>(
     Ok(())
 }
 
-fn parse_bundle<D: serde::de::DeserializeOwned>(bundle: &[Vec<u8>]) -> Result<Vec<D>> {
+fn parse_bundle<D: serde::de::DeserializeOwned>(
+    bundle: &[ethers::prelude::Bytes],
+) -> Result<Vec<D>> {
     bundle
         .iter()
-        .filter(|item| !item.is_empty()) // filter out empty items
-        .map(|item| Ok(bincode::deserialize::<D>(&item)?))
+        .filter(|item| !item.to_vec().is_empty()) // filter out empty items
+        .map(|item| Ok(bincode::deserialize::<D>(&item.to_vec()[..])?))
```

Comment:
Same here, instead of going through a new `Vec`, you might be able to use `&item.as_slice()`